src/components/API.js

```diff
@@ -25,7 +25,7 @@ export default class Api {
       },
     });
   }
-  updateProfileImage() {
+  updateProfileImage(data) {
     return this._request(
       "https://around.nomoreparties.co/v1/group-12/users/me/avatar",
```

Comment:
According to the task you need to pass full `https://around.nomoreparties.co/v1/group-12` into the constructor of `Api`.  as `baseUrl` . See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `this._baseUrl` (not to duplicate the code)